torch/csrc/jit/python/script_init.cpp

```diff
@@ -146,11 +146,11 @@ struct PythonResolver : public Resolver {
   ClassTypePtr classType_;
 };
 
-std::shared_ptr<PythonResolver> pythonResolver(ResolutionCallback rcb) {
+std::shared_ptr<PythonResolver> pythonResolver(const ResolutionCallback& rcb) {
```

Comment:
These don't seem related. Are these in response to suggestions from clint tool?

Reply:
yes, only the below part where i left a comment on is intended change.